src/Libraries/RevitNodes/Elements/PerformanceAdviserRule.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Autodesk.Revit.DB;
+using Autodesk.DesignScript.Runtime;
+using System;
+using RevitServices.Transactions;
+using RevitServices.Persistence;
+using System.Collections.Generic;
+using System.Collections;
+
+
+namespace Revit.Elements
+{
+    /// <summary>
+    /// Performance Adviser Rule
+    /// </summary>
+    public class PerformanceAdviserRule
+    {
+        internal Autodesk.Revit.DB.PerformanceAdviserRuleId InternalId
+        { 
+            get; set;
+        }
+
+        [SupressImportIntoVM]
+        public PerformanceAdviserRule(Autodesk.Revit.DB.PerformanceAdviserRuleId id)
+        {
+            this.InternalId = id;
+        }
+
+        /// <summary>
+        /// The name of the rule.
+        /// </summary>
+        public string Name
+        {
+            get
+            {
+                PerformanceAdviser adviser = PerformanceAdviser.GetPerformanceAdviser();
+                return adviser.GetRuleName(this.InternalId);
+            }
+        }
+
+        /// <summary>
+        /// The description of the rule.
+        /// </summary>
+        public string Description
+        {
+            get
+            {
+                PerformanceAdviser adviser = PerformanceAdviser.GetPerformanceAdviser();
+                return adviser.GetRuleDescription(this.InternalId);
+            }
+        }
+
+        /// <summary>
+        /// Check is the rule is activated
```

Comment:
Since the property is a boolean, can you change the comment to something like: A boolean indicating if the rule is actived. You can move the current comment above the **return**